.gitignore

```diff
@@ -148,7 +148,11 @@ src/stcal/_version.py
 
 # auto-generated API docs
 docs/source/api
+<<<<<<< HEAD
```

Comment:
There are conflict markings from rebase or merge in this file.  This file needs editing to handle the conflicts.

Reply:
I removed the conflict markings.